spectacles/exceptions.py

```diff
@@ -182,6 +182,7 @@ def __init__(
         field_name: str,
         content_type: str,
         title: str,
+        folder: str,
```

Comment:
Haven't we found that folders are sometimes missing? Should this be optional?

Reply:
I don't think so. I can't find anywhere [in the previous version](https://github.com/spectacles-ci/spectacles/blob/v2.2.0/spectacles/exceptions.py#L185) where we considered this optional. I thought there was an issue related to this, but I can't seem to find it.

Reply:
Isn't it this PR? https://github.com/spectacles-ci/spectacles/pull/576

Reply:
Thanks—I thought I remembered this but couldn't find it! I'll make the update.

Reply:
Though it is weird that this line, as a result, wasn't set to Optional. https://github.com/spectacles-ci/spectacles/blob/726f19e4c49431d8feceeb234523a5aca30bfe62/spectacles/exceptions.py#L185